.github/workflows/pull-request-management.yml

```
@@ -268,11 +268,11 @@ jobs:
           - 'example-l2ls-fabric'
           - 'example-single-dc-l3ls'
         ansible_version:
-          - 'ansible-core>=2.11.3,<2.13.0'
+          - 'ansible-core>=2.12.6,<2.15.0,!=2.13.0'
         # Also test minimum ansible version for one scenario.
         include:
           - avd_scenario: 'eos_designs_unit_tests'
-            ansible_version: 'ansible-core==2.11.3'
+            ansible_version: 'ansible-core==2.12.6'
```

I would suggest we also test eos_designs_unit_tests with 2.13.x train.

```suggestion
            ansible_version: 'ansible-core==2.12.6'
          - avd_scenario: 'eos_designs_unit_tests'
            ansible_version: ''ansible-core>=2.13.1,<2.14.0''  
```

---

good idea